integration/core/test_cli.py

```diff
@@ -810,3 +893,8 @@ def test_backup_cli(bin, controller_client, replica_client,
         cleanup_replica(replica_client)
         cleanup_replica(replica_client2)
         cleanup_controller(controller_client)
+        backup_volume_list(bin, controller_client, replica_client,
```

Comment:
You may not need to form a new volume. Put the function in the backup_core() should be easier.